README.md

```diff
@@ -6,16 +6,16 @@
 
 # Quickstart
 
-## Prequisites
+## Prerequisites
 
 1. Download and install [Git](https://git-scm.com/downloads)  
    This will be used to download the repository
 
 2. Download and install [Git LFS](https://git-lfs.github.com/)  
    This will be used to download any required binary assets, such as images or pdfs
 
-3. Download and install [Node](https://nodejs.org/en/download/)  
-   This is the programming lanugage required to run the project
+3. Download and install [Node](https://nodejs.org/en/download/) (choose v18.20.4)  
```

Comment:
nit(non-blocking): not sure how precise we want to be with node version. All our CI uses `18.x`, I personally run `20.7.0` (due to known issues for our codebase on higher https://github.com/IDEMSInternational/open-app-builder/issues/2185), so I think could be fine to just suggest 

> choose v18 or v20

Reply:
Yeh, @kwAsant had some general issues running v22, I'm tempted to specify v18 for now